src/Symfony/Component/Security/Core/Authentication/AuthenticationManagerInterface.php

```diff
@@ -13,12 +13,15 @@
 
 use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
 use Symfony\Component\Security\Core\Exception\AuthenticationException;
+use Symfony\Component\Security\Http\Firewall\X509AuthenticationListener;
 
 /**
  * AuthenticationManagerInterface is the interface for authentication managers,
  * which process Token authentication.
  *
  * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @internal since Symfony 5.3
```

Comment:
Is there something blocking us from deprecating this? Afaik, it's not used by the authenticator implementation

Reply:
It's implemented by the `NoopAuthenticationManager` so deprecating it now causes unfixable notices to end users